src/agrag/utils.py

```diff
@@ -4,7 +4,7 @@
 
 import torch
 
-logger = logging.getLogger("rag-logger")
+logger = logging.getLogger("AutoGluon-RAG-logger")
```

Comment:
could you use a single place for name, so next time you do not need to rewrite for every script.

Reply:
Yeah that's a better idea, will update